src/cow-react/pages/Faq/LimitOrdersFaq.tsx

```diff
@@ -52,12 +52,32 @@ export default function LimitOrderFAQ() {
             <p>CoW Protocol supports Fill or kill as well as Partially fillable limit orders.</p>
             <p>With Fill or kill, the limit order is executed in its entirety, or not at all.</p>
             <p>
-              With Partially fillable limit orders, the order does not need to be executed in full. It may be executed
-              in part, so long as the limit price is respected. This would be relevant, for example, if a{' '}
-              <ExternalLinkFaq href="https://docs.cow.fi/off-chain-services/solvers">solver</ExternalLinkFaq> is able to
-              find liquidity to fill half of an order but not all of it.
+              With Partially fillable limit orders, meanwhile, the order can be filled gradually as liquidity becomes
+              available. Partially fillable orders usually complete faster than Fill or kill orders and also provide
+              better prices, so they are the default mechanism used to execute limit orders on CoW Swap. Users can turn
+              off Partially fillable limit orders and revert back to Fill or kill through the swap panel settings.
```

Comment:
`Fill or kill`, why capitalising only the `P` and the `F` in `Partially fillable` and `Fill or kill`?